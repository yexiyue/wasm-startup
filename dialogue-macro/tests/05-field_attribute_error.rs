use dialogue_macro::Dialogue;

#[derive(Dialogue)]
struct User2 {
    #[dialogue(default = ["wasm-project","wasm-project2"], prompt = "请输入项目名称" options=["wasm-project","wasm-project2",123], multi_select = true)]
    name: String,
    age: i32,
}

fn main() {}
