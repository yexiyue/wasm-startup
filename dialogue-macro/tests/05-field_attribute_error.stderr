error: expected `,`
 --> tests/05-field_attribute_error.rs:5:79
  |
5 |     #[dialogue(default = ["wasm-project","wasm-project2"], prompt = "请输入项目名称" options=["wasm-project","wasm-project2",123], multi_sel    ...
  |                                                                                      ^^^^^^^
